README.md

```diff
@@ -1,2 +1,25 @@
-# panoptes-net-client
+# Panoptes .NET Client
 A .NET client for accessing the Panoptes API
+
+## General
+This repo contains three projects: a `ConsoleApp` (to test calls while developing), the `PanoptesNetClient`, and a separate `PanoptesNetClientTests` project to cover testing.
+
+## Requirements
+In order to run this client, an app must have a config transform with two config files (debug and release) to account for the Config class calling the `ConfigurationManager`.
```

Comment:
Will the config files ship with this package? If not, I think you should add some explanation on how they will be formatted

Reply:
They will not ship with this package. Config files are usually placed on the app instead of an extension. I'll add to the readme to reflect this.